docs/manage/_content.rst

```diff
@@ -178,6 +184,30 @@ If Kolibri is installed on an older or a low-resource device, you can try the fo
 On Linux and OSX you will need to enable the **Show hidden folders** option in order to view the ``.kolibri`` folder.    
 
 
+Update Content Channels
+-----------------------
+
+Public channels on Kolibri Studio are periodically updated from their original sources. Private and unlisted channels that you have on you local Kolibri device may have also changed since you first imported them, and have some new content added in the meantime. You can update your local channels from Kolibri Studio if you have internet connection, from another device in your local network where the content has already been updated, or from an external drive where the latest version of the channels has been :ref:`previously exported <export>`.
```

Comment:
```suggestion
Public channels on Kolibri Studio are periodically updated from their original sources. Private and unlisted channels that you have on your local Kolibri device may also have changed since you first imported them. You can update your local channels from Kolibri Studio from online sources, from another updated device in your local network or from an external drive where the latest version of the channels has been :ref:`previously exported <export>`.
```